docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -208,6 +208,12 @@
   to learn more about custom data, and how it can be used to influence the
   provisioning process.
 
+- `user_data_file` (string) - Specify a file containing custom data to inject into the cloud-init
```

Comment:
I think this is generated? Otherwise: `custom data` -> `user data` for consistency

Reply:
yep, auto generated, once I update the other it should be fixed!